tests/integration/helpers/includes-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Helper | includes', function(hooks) {
+  setupRenderingTest(hooks);
+
+  // Replace this with your real tests.
+  test('it renders', async function(assert) {
+    this.set('inputValue', '1234');
+
+    await render(hbs`{{includes inputValue}}`);
+
+    assert.equal(this.element.textContent.trim(), '1234');
```

Comment:
can we make this a real test? and regarding my other comment, we can add a case that tests if you pass in undefined and make sure it doesn't explode 😂